api/cpp/cbindgen.rs

```diff
@@ -604,11 +605,18 @@ fn gen_corelib(
     friend inline LayoutInfo operator+(const LayoutInfo &a, const LayoutInfo &b) { return a.merge(b); }
     friend bool operator==(const LayoutInfo&, const LayoutInfo&) = default;".into(),
     );
+    config.export.body.insert(
+        "WindowEvent".to_owned(),
+        "/* Some members of the WindowEvent enum have destructors (with SharedString), but thankfully we don't use these so we can have an empty constructor */
+    ~WindowEvent() {}"
```

Comment:
Because the key events holds a SharedString, C++ deletes the destructor by default.  Here I just implement it, but it is going to leak types with destructor such as SharedString, if such union member are used.